src/content/doc-sdk-dotnet/index.mdx

```diff
@@ -11,18 +11,19 @@ import Version from '@components/Version.astro';
 
 # .NET SDK
 
-The [SurrealDB SDK for .NET](https://nuget.org/packages/surrealdb.net) enables simple and advanced querying of a remote database from client or server-side code. Remote connections automatically reconnect when the connection is terminated.
+The SurrealDB SDK for C# and .NET is the primary SDK for interacting with SurrealDB from server-side applications, systems, and APIs, allowing you to integrate SurrealDB into your website or application backend, and serve dynamic content to your users. You can use the .NET SDK to interact with your SurrealDB database instances, or to run SurrealDB as an embedded database within your C# application, with functionality for executing queries, managing data, running database functions, authenticating to the database, building user signup and authentication functionality, and subscribing to data changes with live queries. When connecting to remote database instances, connections automatically reconnect when terminated.
 
-To contribute to this documentation, submit an Issue or Pull Request in the [`docs.surrealdb.com`](https://github.com/surrealdb/docs.surrealdb.com) repository.
-
-To contribute to the SDK code, submit an Issue or Pull Request in the [`surrealdb.net`](https://github.com/surrealdb/surrealdb.net) repository.
+> [!IMPORTANT]
+> The SDK requires .NET version `5.0` or greater, or .NET Core `3.0` or higher, and is available as a [nuget package](https://www.nuget.org/packages/SurrealDb.Net).
```

Comment:
todo: well, this sentence is not really true. It supports .NET 6 or higher AND .NET Standard 2.1, which englobe .NET 5+, .NET Core 3.X, Mono, Xamarin and Unity.

There is a table here https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-1#select-net-standard-version

Maybe we can refer to it or list the ones in the table? (except for .NET Framework and UWP of course)